src/main/java/com/commercetools/project/sync/SyncerFactory.java

```diff
@@ -69,6 +62,9 @@ public static SyncerFactory of(
                 .sync(runnerNameOptionValue, isFullSync)
                 .toCompletableFuture(),
             TypeSyncer.of(sourceClient, targetClient, clock)
+                .sync(runnerNameOptionValue, isFullSync)
+                .toCompletableFuture(),
+            StateSyncer.of(sourceClient, targetClient, clock)
```

Comment:
`StateSyncer` is added into the list `typeAndProductTypeSync`. Shall we change the variable name here to avoid misunderstanding?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133/commits/09023e85db4080b67cbb907aa5bda6c7e29ae4bd